README.md

```diff
@@ -47,20 +48,33 @@ stringData:
 apiVersion: v1
 kind: Secret
 metadata:
-  name: your-db-secret
+  name: db-instance-secret-name
 type: Opaque
 stringData:
   password: password-for-the-database-instance
   ssh_public_key: SSH-PUBLIC-KEY
 
 ```
 
-2. To create instances of custom resources (provision databases), edit [ndb_v1alpha1_database.yaml](config/samples/ndb_v1alpha1_database.yaml) file with the NDB installation and database instance details and run:
+Apply the secrets:
+
+```
+kubectl apply -f <path/to/secrets.yaml>
+
+```
+You can optionally verify that they have been created:
+
+```sh
+kubectl get secrets
+```
+
+2. To create an instances of CR (provision databases), update the NDB Server IP in [ndb_v1alpha1_database.yaml](config/samples/ndb_v1alpha1_database.yaml) in spec section. Ensure that the names of the secrets provided in the ndb_v1alpha1_database.yaml exactly match with the names of secrets defined in the secrets.yaml
```

Comment:
Since we're getting verbose in this point, can you include the clusterId and other details too?